test/assets/unrunnableTestCases.csv

```diff
@@ -4775,42 +4749,42 @@ d1e69567,Error: XPST0017: Function parse-ietf-date with arity of 1 not registere
+d1e78807k,Error: XPST0003: Unable to parse: "xquery version "3.1"; module namespace prod = "http://datypic.com/prod"; declare context item as element(catalog) external; declare variable $prod:label as xs:string external; declare variable $prod:prods as element(product)* := product; declare function prod:countProds ($prods as element(product)*) as xs:string   {concat($prod:label, ': ', count($prods))}; declare function prod:countProds ($prods as element(product)*,                                    $dept as xs:string) as xs:string   {concat($prod:label, ': ', count($prods[@dept = $dept]))};". Expected "%", "(", "Q", "array", "attribute(", "attribute()", "comment()", "document-node(", "element", "function", "item()", "map", "namespace-node()", "node()", "processing-instruction(", "processing-instruction()", "schema-attribute(", "schema-element", "text()", [A-Z_a-zÀ-ÖØ-öø-˿Ͱ-ͽͿ-῿‌‍⁰-↏Ⰰ-⿯、-퟿豈-﷏ﷰ-�], or [�-�] but " " found. xquery version "3.1"; module namespace prod = "http://datypic.com/prod"; declare context item as[Error is around here] element(catalog) external; declare variable $prod:label as xs:string external; declare variable $prod:prods as element(product)* := product; declare function prod:countProds ($prods as element(product)*) as xs:string   {concat($prod:label, ': ', count($prods))}; declare function prod:countProds ($prods as element(product)*,                                    $dept as xs:string) as xs:string   {concat($prod:label, ': ', count($prods[@dept = $dept]))};
 XMark-Q3,Error: Not implemented: Intermediate clauses in flwor expressions are not implemented yet
 XMark-Q4,Error: Not implemented: Intermediate clauses in flwor expressions are not implemented yet
 XMark-Q5,Error: Not implemented: Intermediate clauses in flwor expressions are not implemented yet
```

Comment:
I expected more succeeding tests. 26 new success is less then I expected

Reply:
I enabled the TypeSwitch tests in another blacklist we have. This causes more 'new' tests to succeed by running 144 previously not running tests!